modules/common/react/spec/Popper.spec.tsx

```diff
@@ -1,104 +1,79 @@
-jest.mock('react-dom', () => ({
-  createPortal: jest.fn().mockImplementation(children => children),
-}));
-
-// https://github.com/FezVrasta/popper.js#how-to-use-popperjs-in-jest
-jest.mock('popper.js', () => {
-  return jest.fn().mockImplementation(() => {
-    return {
-      destroy: () => {}, // eslint-disable-line no-empty-function
-    };
-  });
-});
+/// <reference types="@testing-library/jest-dom/extend-expect" />
+
+import React from 'react';
+import {render, getByTestId} from '@testing-library/react';
```

Comment:
The Portal mocking that was done here and the Popper changes broke these tests in ways that I didn't understand. I rewrote the tests using React Testing Library instead and didn't need any Portal hacks. 🤷‍♂